web-api/src/business/useCases/auth/signUpUserInteractor.ts

```diff
@@ -28,6 +28,11 @@ export const signUpUserInteractor = async (
       email: user.email,
     });
 
+  // Note that this check can fail to catch two (nearly) simultaneous requests,
+  // and Cognito can therefore create accounts with the same email.
+  // (See https://stackoverflow.com/questions/50730759/user-pool-allows-two-users-with-same-email-despite-configuration)
+  // On the frontend, we can prevent multiple submissions; if we wanted to enforce non-duplicates on the back-end,
```

Comment:
It bothers me that I used both `frontend` (unhyphenated) and `back-end` (hyphenated). I'd fix this before merging to staging, if this comment is even still around.